contest/contest_info.py

```diff
@@ -99,6 +103,72 @@ def get_scoreboard(contest):
 
     return scoreboard
 
+def get_scoreboard_csv(contest_id):
```

Comment:
Is that possible to let user choose which version of scoreboard(penalty or testcase) they want to download?